client/src/VideoDisplay.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Link } from 'react-router-dom';
+
+
+
+
+const VideoDisplay = ({ video, handleUpVote, handleDownVote, handleRemoveVideo }) => {
+return (
+<div className='video-card'>
+<h2>{video.title}</h2>
+<iframe
+width="560"
+height="315"
+src={`https://www.youtube.com/embed/${video.url.split('v=')[1]}`}
```

Comment:
What would happen if the video URL didn't contain `v=`? Is there a way we could handle that better?